src/libs/API/parameters/UpdateNetSuiteSyncTaxConfigurationParams.ts

```diff
@@ -0,0 +1,6 @@
+type UpdateNetSuiteSyncTaxConfigurationParams = {
+    policyID: string;
+    enabled: boolean;
```

Comment:
@yuwenmemon Even if I don't pass `enabled` in the API call it succeeds. Is that expected? Unlikely to be optional from UI, but still confirming.